src/api/api.js

```diff
@@ -31,8 +37,26 @@ export const getStockHistory = async params => await axios.get(`${urls.charts}/$
  * action
  * portfolioId
  */
-export const createTransaction = async params => await axios.post(urls.transactions, params, getRequestConfig());
-
-export const registerUser = async (userInfo) => {
-  await axios.post('http://localhost:5005/api/v1.0/users', userInfo);
```

Comment:
I don't know if you want to get rid of the registerUser function here... Did you pull before you did this pr?

Reply:
hmmm that's not good...